src/main/scala/org/broadinstitute/dsde/firecloud/webservice/PassthroughApiService.scala

```diff
@@ -25,6 +25,7 @@ trait PassthroughApiService extends Directives with StreamingPassthrough {
     pathPrefix("version" / "executionEngine")(streamingPassthrough(s"$rawls/version/executionEngine")),
     // Sam
     pathPrefix("api" / "proxyGroup")(streamingPassthrough(s"$sam/api/google/user/proxyGroup")),
+    pathPrefix("register" / "user")(streamingPassthrough(s"$sam/register/user")),
     pathPrefix("register")(streamingPassthrough(s"$sam/register/user")),
```

Comment:
This is ugly, but it allows:
| Orch | Sam |
| /register | /register/user |
| /register/user/v1/termsofservice | /register/user/v1/termsofservice |